docs/en_us/shared/preface.rst

```diff
@@ -13,14 +13,83 @@ goals, and solve problems, edX provides a variety of information resources.
 
 To help you find what you need, browse the edX offerings in these categories:
 
-* :ref:`Resources for Course Teams`
-* :ref:`Resources for Researchers`
-* :ref:`Resources for Developers`
-* :ref:`Resources for Students`
+* `The edX Partner Portal`_
+* `The Open edX Portal`_
+* `Release Announcements through the Open edX Portal`_
+* `Resources for Course Teams`_
+* `Resources for Researchers`_
+* `Resources for Developers`_
+* `Resources for Students`_
 
 All members of the edX community are encouraged to make use of any of the
 resources described in this preface.
 
+.. _The edX Partner Portal:
+
+***********************
```

Comment:
This is really great information about the Partner Portal. I wonder if you could remove a lot of the rest of this page since the information is now duplicated there. This page is a little overwhelming as it tries to address all sorts of unrelated topics (creating courses, guide to students, resources for researchers etc).